packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1046,10 +1056,14 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   }
 
   /// {@template flutter.rendering.RenderEditable.paintCursorAboveText}
-  /// If the cursor should be painted on top of the text or underneath it.
+  /// If the cursor and the text selection highlight should be painted on top of the text or underneath it.
```

Comment:
Nit: I think this line is too wide.

Reply:
Sorry i don't understand what you mean🤣; Could you give me more information?

Reply:
Oh sorry, I just meant that the line should be about 80 characters, so you should split some of this text onto a new line.